quipucords/fingerprinter/jboss_fuse.py

```diff
@@ -71,6 +68,10 @@ def get_version(eap_version):
     return new_versions
 
 
+def _get_val_or_empty_list(data_dict, key):
```

Comment:
Maybe put this in a common file with the other helper methods to DRY it up

Reply:
Oh I didn't comment on this but I added a DRY solution in a project level `utils` module.

Reply:
Replaced this function with the helper "default_getter" https://github.com/quipucords/quipucords/blob/96b821a58a7291c883d007c49e86f49418622bfb/quipucords/fingerprinter/jboss_fuse.py#L84-L101